src/test/java/com/issoft/training/tests/AutomationPracticeShoppingTests.java

```diff
@@ -0,0 +1,106 @@
+package com.issoft.training.tests;
+
+import com.issoft.training.base.TestBase;
+import com.issoft.training.config.ConfProperties;
+import com.issoft.training.listener.AllureListener;
+import com.issoft.training.pages.*;
+import com.issoft.training.utility.Constants;
+import io.qameta.allure.*;
+import org.testng.Assert;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+
+@Listeners({ AllureListener.class })
+@Epic("Regression tests")
+@Feature(value = "Automation practice - Shopping tests")
+public class AutomationPracticeShoppingTests extends TestBase {
+    // page objects
+    CreateAccountPage createAccountPage;
+    AccountInformationPage accountInformationPage;
+    LoginAccountPage loginAccountPage;
+    MyAccountPage myAccountPage;
+    MyWishlistsPage myWishlistsPage;
```

Comment:
You can initialize pages there (on class level). It will allow you to remove repeated initialization from methods